roles/openstack_overcloud/tasks/main.yml

```diff
@@ -1,6 +1,11 @@
+- name: slurp OpenStack release file
+  slurp:
+    src: /etc/rhosp-release
+  register: rhosp_release
+
 - name: Set fact for OpenStack release
   set_fact:
-    stockpile_osp_overcloud_release: "{{ lookup('file', '/etc/rhosp-release') }}"
+    stockpile_rhosp_overcloud_release: "{{ rhosp_release['content'] | b64decode  }}"
```

Comment:
Any reason why this can't be left as it was earlier ? 
Because this change will mean that existing data in elasticsearch would've all docs referring to release as osp_overcloud_release while new ones would have rhosp_overcloud_release.

Reply:
@aakarshg Lookup works only on the ansible controller but fails to query on the other nodes

Reply:
Okay and also the above comment i'm referring to name of the var.

Reply:
Have updated with previous name